[bazel] New: added support for bazel options (#35)
endorlabs/github-action: README.md

| `scan_secrets` | Scan source code repository and generate findings for secrets. See also `scan_git_logs`. (Default: `false`) |
| `scan_summary_output_type` | Set the desired output format to `table`, `json`, `yaml`, or `summary`. (Default: `json`) |
| `tags` | Specify a list of user-defined tags to add to this scan. Tags can be used to search and filter scans later. |
| `use-bazel` | Enable the usage of Bazel for the scan. (Default: `false`)|

> | `bazel_exclude_targets` | The list of Bazel targets that are not to be scanned. |

Reviewer: How about calling these options `bazel_targets_include` and `bazel_targets_exclude` to match `bazel_targets_query`?

Author: Actually this is actually on purpose;
- include and exclude ends with targets since we expect a list of targets.
- query is at the end because we expect a query.

My goal was that following the semantic of the flags, users could expect what input it needs.

> | `bazel_exclude_targets` | The list of Bazel targets that are not to be scanned. |

Reviewer: ```suggestion
| `bazel_targets_exclude` | Specify a a list of Bazel targets to exclude from scan. |
```

> | `bazel_include_targets` | The list of Bazel targets that are to be scanned. If not given, Endorctl will by default used the targets queried from --bazel-targets-query. If given ---bazel-targets-query will be ignored. |

Reviewer: ```suggestion
| `bazel_targets_include` | Specify a list of Bazel targets to scan. If `bazel_targets_include` is not set the `bazel_targets_query` value is used to determine with bazel targets to scan. |
```

> | `bazel_targets_query` | The bazel query used by Endorctl to discover the targets to scan. (Default: `kind("(py_binary|java_binary)", //...)`) |

Reviewer: ```suggestion
| `bazel_targets_query` | Specify a bazel query to determine with Bazel targets to scan. Ignored if `bazel_targets_include` is set. (Default: `kind("(py_binary|java_binary)", //...)`) |
```
